src/system.c

```diff
@@ -824,8 +831,10 @@ void sysLaunchLoaderElf(const char *filename, const char *mode_str, int size_cdv
 
 #ifdef __DECI2_DEBUG
     modules |= CORE_IRX_DECI2 | CORE_IRX_ETH;
-#elif defined(__INGAME_DEBUG)
+#elif defined(__INGAME_DEBUG) && !defined(TTY_PPC_UART)
```

Comment:
logic looks too complex. Isn't it easier to check EE_SIO and PPC_TTY only? So to check here if it is necessary to NOT load ETH, and for all others to load ETH?